gdsfactory/port.py

```diff
@@ -1046,7 +1046,24 @@ def auto_rename_ports_layer_orientation(
 if __name__ == "__main__":
     import gdsfactory as gf
 
-    c = gf.c.straight()
-    p2 = c["o2"]
-    p2.x = 20
+    my_cross_section = gf.CrossSection(sections=[gf.Section(layer=(1, 1), width=2)])
```

Comment:
Should this be added as a test so that CI notices if it breaks?

Reply:
yes, we can do that